packages/main/src/TableRow.ts

```diff
@@ -114,6 +115,29 @@ class TableRow extends TableRowBase {
 	@property({ type: Boolean })
 	movable = false;
 
+	_dragStartHandler: (e: DragEvent) => void;
+	_dragEndHandler: (e: DragEvent) => void;
+
+	constructor() {
+		super();
+		this._dragStartHandler = this._ondragstart.bind(this);
+		this._dragEndHandler = this._ondragend.bind(this);
+	}
+
+	onEnterDOM() {
+		super.onEnterDOM();
+
+		this.addEventListener("dragstart", this._dragStartHandler);
+		this.addEventListener("dragend", this._dragEndHandler);
+	}
+
+	onExitDOM() {
+		super.onExitDOM();
+
+		this.removeEventListener("dragstart", this._dragStartHandler);
+		this.removeEventListener("dragend", this._dragEndHandler);
```

Comment:
why there are changes in the Table component and we are not set as reviewer?

We have a event delegation system in the table. please add dragstart and dragend to the list and remove all these lines. 
https://github.com/SAP/ui5-webcomponents/blob/main/packages/main/src/Table.ts#L407

As convention _ondragstart and _ondragend handlers below will be called.

BTW, as another convention, no test no +2 sorry...

Reply:
Hello @aborjinik, this change is still work in progress.

When the dragged element is in a shadow dom, the global dragstart handling `document.body.addEventListener("dragstart", ondragstart);` doesn't work, as the target is the whole web component, but not the element inside the shadow dom. I'll check your suggestions and when we're ready, I'll add you as a reviewer.